src/v/cluster/tests/replicas_rebalancing_tests.cc

```diff
@@ -0,0 +1,92 @@
+
+#include "cluster/metadata_cache.h"
+#include "cluster/tests/rebalancing_tests_fixture.h"
+#include "model/metadata.h"
+#include "model/namespace.h"
+#include "test_utils/async.h"
+
+#include <absl/container/node_hash_map.h>
+#include <boost/test/tools/old/interface.hpp>
+
+#include <algorithm>
+
+absl::node_hash_map<model::node_id, size_t>
+calculate_replicas_per_node(const cluster::metadata_cache& cache) {
+    absl::node_hash_map<model::node_id, size_t> ret;
+
+    for (auto& tp_md : cache.all_topics_metadata()) {
+        if (tp_md.tp_ns.ns == model::redpanda_ns) {
```

Comment:
should we also reject internal namespace here? I thought I saw that we did earlier for rebalancing logic. maybe this is different scenario..

Reply:
this is just a test, but good point we should be consistent, I will fix this.

Reply:
ah yeh I didn't notice it was a test. nbd